daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala

```diff
@@ -145,6 +145,26 @@ private[archive] class DecodeV1(minor: LV.Minor) extends Decode.OfPackage[PLF.Pa
       val defs = mutable.ArrayBuffer[(DottedName, Definition)]()
       val templates = mutable.ArrayBuffer[(DottedName, Template)]()
 
+      if (!onlySerializableDataDefs) {
+        // collect type synonyms
+        lfModule.getSynonymsList.asScala
```

Comment:
```suggestion
        assertSince(LV.Feature.typeSynonyms) 
        lfModule.getSynonymsList.asScala
```

assuming we add in `daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala` the following line:

``
    val typeSynonyms = v1_dev
``

Reply:
ok, I'll have a look at what needs doing here...

Reply:
This is done now. Thank you.